src/Compilers/CSharp/Test/Emit/PrivateProtected.cs

```diff
@@ -266,14 +268,15 @@ void M()
     }
 }
 ";
-            CreateCompilation(source, parseOptions: TestOptions.Regular7_2)
+            CreateCompilation(source, parseOptions: TestOptions.Regular7_2, assemblyName: "Paul")
                 .VerifyDiagnostics(
-                // (12,11): error CS0122: 'Base.Field1' is inaccessible due to its protection level
+
```

Comment:
This looks wrong. There are no IVT attributes or references in this compilation.

Reply:
b.Field1 is trying to access a private protected field it has no access to. What error should be thrown instead?

Reply:
The old message looks correct here. The current one doesn't make any sense. How could a compilation provide IVT access to itself?